int8 headerID
int8 ID_UAS_Type
string UAS_ID

int8 headerLocation
int8 Status_Flags
uint8 Track_Direction
uint8 Speed
int8 Vertical_Speed
int32 Latitude
int32 Longitude
int16 Pressure_Altitude
uint16 Geodetic_Altitude
int16 Height
int8 Hori_Vert_Accuracy
int8 Baro_Speed_Accuracy
uint16 TimeStamp
int8 TimeStamp_Accuracy